src/proxy/outbound.rs

```diff
@@ -79,7 +79,8 @@ impl Outbound {
         let (sub_drain_signal, sub_drain) = drain::channel();
         let pool = proxy::pool::WorkloadHBONEPool::new(
             self.pi.cfg.clone(),
-            self.enable_orig_src,
+            // Do not need to spoof with inpod mode for outbound
+            self.enable_orig_src && !self.pi.cfg.inpod_enabled,
```

Comment:
Can we just do this check in `new` and set `enable_orig_src = false` once-for-all?

Reply:
good call, done